### DISCUSSION TITLE
Why Reranker works when it is hybrid search?

### DISCUSSION THREAD

**Discussion by tjtanaa** [lancedb/lancedb#1095] [Q&A]
May I know why we can only invoke reranker when it is a hybrid search, where we need both indices: vector and full-text-search indices?

iirc, reranker can also be applied to the queries after vector search without the need of full-text-search. What is the benefit of current design compared to the methond aforementioned?

**Comment by AyushExel** [comment]
We're working on supporting reranking with vector only and fts only queries as well. I plan to get to it within this week.
